apps/desktop/src/renderer/components/sync/ConflictResolver.module.css

```
@@ -1,285 +1,268 @@
-  cursor: pointer;
-  transition: all 0.2s;
-}
-
-.keepButton:hover:not(:disabled) {
-  background: var(--accent-hover);
-}
-
-.keepButton:disabled {
-  opacity: 0.5;
-  cursor: not-allowed;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)